README.md

```diff
@@ -160,6 +160,56 @@ int install_hooks()
 
 ```
 
+Example - Using Python ctypes
+-----------------------------
+```
```

Comment:
@eli-fin Thanks.
Could you replace this line as below for [syntax highlighting](https://docs.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks#syntax-highlighting)?

    ```python

Reply:
Yep. Missed that :)